R/functions.R

```diff
@@ -1181,8 +1181,11 @@ test_gene_enrichment_bulk_EGSEA <- function(.data,
 																							 .entrez,
 																							 .abundance = NULL,
 																							 .contrasts = NULL,
-																						method,
+																						     method,
 																							 species,
+																						     msigdb.gsets,
+																						     kegg.exclude,
+																						     min.size,
```

Comment:
As for the arguments I would keep it simpler:
- a self explanatory one (style tidyverse), what about `gene_collections`?
- I would keep one argument for the user. So gene_collections can include or not "kegg", and in the backend we sort this out.

Reply:
ok sounds good, I'll see if I can do that.

Reply:
done and pushed here now